src/elements/content-sidebar/Sidebar.js

```diff
@@ -114,6 +116,10 @@ class Sidebar extends React.Component<Props, State> {
             this.setState({ isDirty: true });
         }
 
+        if (prevAnnotation !== annotation) {
```

Comment:
Should we check that `annotation` is defined, as well?

Reply:
Good point